libs/dart/didcomm/atm_client/lib/src/messages/atlas/update_mediator_instance_deployment_message.dart

```diff
@@ -19,14 +18,13 @@ class UpdateMediatorInstanceDeploymentMessage extends PlainTextMessage {
     super.body = const {},
     super.threadId,
   }) : super(
-          type: Uri.parse(
-            'affinidi.io/operations/ama/updateMediatorInstanceDeployment',
-          ),
+          operationName: 'updateMediatorInstanceDeployment',
```

Comment:
Same here, let's extract into variable so you can also make use of it in [here](https://github.com/affinidi/affinidi-tdk/pull/1077/files#diff-c98c9eb90764466f46248e51fa5ee726b48560cbb1aadf273525ecdd3a7f3a35R38).